src/vscode-bicep/src/test/e2e/runTests.ts

```diff
@@ -11,37 +13,57 @@ import {
 
 async function go() {
   try {
-    const vscodeExecutablePath = await downloadAndUnzipVSCode("stable");
-    const cliPath = resolveCliPathFromVSCodeExecutablePath(
-      vscodeExecutablePath
+    // Do not import the json file directly because it's not under /src.
+    // We also don't want it to be included in the /out folder.
+    const packageJsonPath = path.resolve(__dirname, "../../../package.json");
+    const packageJson = JSON.parse(
+      fs.readFileSync(packageJsonPath, { encoding: "utf-8" })
     );
+    const minSupportedVSCodeSemver = minVersion(packageJson.engines.vscode);
 
-    const isRoot = os.userInfo().username === "root";
-
-    // some of our builds run as root in a container, which requires passing
-    // the user data folder relative path to vs code itself
-    const userDataDir = "./.vscode-test/user-data";
-    const userDataArguments = isRoot ? ["--user-data-dir", userDataDir] : [];
-
-    const extensionInstallArguments = [
-      "--install-extension",
-      "ms-dotnettools.vscode-dotnet-runtime",
-      ...userDataArguments,
-    ];
-
-    // Install .NET Install Tool as a dependency.
-    cp.spawnSync(cliPath, extensionInstallArguments, {
-      encoding: "utf-8",
-      stdio: "inherit",
-    });
-
-    await runTests({
-      vscodeExecutablePath,
-      extensionDevelopmentPath: path.resolve(__dirname, "../../.."),
-      extensionTestsPath: path.resolve(__dirname, "index"),
-      extensionTestsEnv: { NODE_ENV: "test" },
-      launchArgs: ["--enable-proposed-api", ...userDataArguments],
-    });
+    if (!minSupportedVSCodeSemver) {
+      throw new Error(
+        "Ensure 'engines.vscode' is properly set in package.json"
+      );
+    }
```

Comment:
FYI you can just `import` the JSON file directly with type safety - I'm doing this here in the playground: https://github.com/Azure/bicep/blob/main/src/playground/webpack.config.ts#L4

Reply:
Yeah I know. Actually that's what I did initially in my first commit. However importing the JSON file directly has a side effect that it will change `rootDir`, because all imported json files are considered source files and will be copied to /out. After compilation, /out will look like this
```
/out
|--/src
  |--/language
  |--extension.ts
  |--...
|--package.json
```
while what we want is
```
/out
|--/language
|--extension.ts
|--...
```

I've added a comment to explain why we don't want to import it directly.

Reply:
Makes sense, thanks for the explanation!